addons/libs/extdata.lua

```diff
@@ -1972,88 +1972,88 @@ end
 
 function decode.AssaultLog(itemid, str) 
     local missions = T{
-        [2491] = T{
+        [2491] = {
             -- "Leujaoam Log"
-            [1] = "Leujaom Cleansing",
-            [2] = "Orichalcum Survey",
-            [3] = "Escort Professor Chanoix",
-            [4] = "Shanarha Grass Conservation",
-            [5] = "Counting Sheep",
-            [6] = "Supplies Recovery",
-            [7] = "Azure Experiments",
-            [8] = "Imperial Code",
-            [9] = "Red Versus Blue",
-            [10] = "Bloody Rhondo"
-    
+            [1] = {name="Leujaom Cleansing"},
```

Comment:
Would change this back to being raw strings, and assemble the `name` attribute later. Also, please move the `missions` table outside of the function and close over it, like this:
```lua
do
    local missions = {
        [2491] = { name = 'Leujaoam',
            [1] = 'Leujaoam Cleansing', -- also fix Leujaoam in this name please
            [2] = 'Orichalcum Survey',
            ...
        },
    }

    function decode.AssaultLog(item_id, str)
        -- use the `missions` variable here
    end
end
```